Readme.md

```diff
@@ -23,7 +23,7 @@ For detailed installation information, see the [wiki](https://github.com/Automat
 
 OS | Command
 ----- | -----
-macOS | Using [Homebrew](https://brew.sh/):<br/>`brew install pkg-config cairo pango libpng jpeg giflib librsvg pixman`
+macOS | Using [Homebrew](https://brew.sh/):<br/>`brew install pkg-config cairo pango libpng jpeg giflib librsvg pixman python@3.12 python-setuptools`
```

Comment:
Yes on `python-setuptools`, but I think python 3.13, 3.11 and 3.10 work as well. I just added python-setuptools to the README for now...